Upstream: no, upstreaming makes only sense if cmake's module is fixed upstream
          (our module is patched with
          http://www.cmake.org/Bug/file_download.php?file_id=5010&type=bug).

From 78a9ad27a94228e68a1700670eb9ad69b8d2ddcd Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Tue, 10 Dec 2013 13:45:00 +0100
Subject: [PATCH] Use cmake's FindFreetype.cmake module.

This fixes building with freetype>=2.5.1.
---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 27debbc..dcc65f7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -372,8 +372,8 @@ ENDIF(NOT PODOFO_BUILD_LIB_ONLY)
 
 FIND_PACKAGE(OpenSSL)
 
-FIND_PACKAGE(FREETYPE REQUIRED)
-MESSAGE("Found freetype library at ${FREETYPE_LIBRARIES}, headers ${FREETYPE_INCLUDE_DIR}")
+FIND_PACKAGE(Freetype REQUIRED)
+MESSAGE("Found freetype library at ${FREETYPE_LIBRARIES}, headers ${FREETYPE_INCLUDE_DIRS}")
 
 FIND_PACKAGE(LIBSTLPORT)
 SET(stlport_libraries_if_use_stlport)
@@ -471,7 +471,7 @@ INCLUDE_DIRECTORIES(
     ${PoDoFo_BINARY_DIR}
     ${PoDoFo_SOURCE_DIR}
     ${PoDoFo_SOURCE_DIR}/src
-    ${FREETYPE_INCLUDE_DIR}
+    ${FREETYPE_INCLUDE_DIRS}
     ${ZLIB_INCLUDE_DIR}
     ${EXTRA_INCLUDES}
      )
-- 
1.8.5.1

